surprising 'incompatible types in assignment' error

```java
import javax.annotation.Nullable;

class Test<T> {
  @Nullable T t;
  Test(@Nullable T t) {
    this.t = t;
  }
}
```

```
$ ~/jsr308/checker-framework-1.9.5/checker/bin/javac -processor org.checkerframework.checker.nullness.NullnessChecker -version Test.java
javac 1.8.0-jsr308-1.9.5
F.java:5: error: [assignment.type.incompatible] incompatible types in assignment.
    this.t = t;
             ^
  found   : T extends @Initialized @Nullable Object
  required: T extends @Initialized @Nullable Object
1 error
```

This is an interesting error.  It seems to occur only when the javax.annotation.Nullable is used (and not the Checker Framework nullable).

What's also interesting is that when we typecheck using the commandline arguments:
`-AprintVerboseGenerics -AprintAllQualifiers`

We get the following message:
```
/Users/jburke/Documents/tmp/Issue471.java:7: error: [assignment.type.incompatible] incompatible types in assignment.
    this.t = t;
             ^
  found   : T[ extends @Initialized @Nullable Object super @Initialized @Nullable Void]
  required: T[ extends @Initialized @Nullable Object super @Initialized @NonNull Void]
```

This seems like it might be an issue with our annotation aliasing mechanism, which maps common annotations to Checker Framework annotations.

--- a/checker/tests/nullness/Issue471.java
+++ b/checker/tests/nullness/Issue471.java
@@ -1,7 +1,6 @@
 // Test case for Issue 471
 // https://github.com/typetools/checker-framework/issues/471
 // javacheck -processor nullness Issue471.java -AprintVerboseGenerics -AprintAllQualifiers
-//@skip-test
 import javax.annotation.Nullable;
 
 class Issue471<T> {

--- a/checker/tests/nullness/generics/Issue783b.java
+++ b/checker/tests/nullness/generics/Issue783b.java
@@ -1,7 +1,6 @@
 // Test case for issue #783:
 // https://github.com/typetools/checker-framework/issues/783
 
-// @skip-test until the issue is fixed
 
 // Note that Issue783a.java, which differs only in the import statement,
 // suffers no type-checking warning.

--- a/framework/src/org/checkerframework/framework/util/element/TypeVarUseApplier.java
+++ b/framework/src/org/checkerframework/framework/util/element/TypeVarUseApplier.java
@@ -8,6 +8,7 @@ import org.checkerframework.framework.type.AnnotatedTypeMirror.AnnotatedTypeVari
 import org.checkerframework.framework.type.ElementAnnotationApplier;
 import org.checkerframework.javacutil.ErrorReporter;
 
+import static org.checkerframework.framework.util.element.ElementAnnotationUtil.addAnnotationsFromElement;
 import static org.checkerframework.framework.util.element.ElementAnnotationUtil.annotateViaTypeAnnoPosition;
 import static org.checkerframework.framework.util.element.ElementAnnotationUtil.contains;
 import static org.checkerframework.framework.util.element.ElementAnnotationUtil.getTypeAtLocation;
@@ -104,6 +105,7 @@ public class TypeVarUseApplier {
      * then applies the explicit annotations written on the type variable
      */
     public void extractAndApply() {
+        addAnnotationsFromElement(typeVariable, useElem.getAnnotationMirrors());
 
         // apply declaration annotations
         ElementAnnotationApplier.apply(typeVariable, declarationElem, typeFactory);

